docs/libcurl/opts/CURLOPT_DEBUG_CONNCACHE.3

```diff
@@ -0,0 +1,98 @@
+.\" **************************************************************************
+.\" *                                  _   _ ____  _
+.\" *  Project                     ___| | | |  _ \| |
+.\" *                             / __| | | | |_) | |
+.\" *                            | (__| |_| |  _ <| |___
+.\" *                             \___|\___/|_| \_\_____|
+.\" *
+.\" * Copyright (C) Daniel Stenberg, <daniel@haxx.se>, et al.
+.\" *
+.\" * This software is licensed as described in the file COPYING, which
+.\" * you should have received as part of this distribution. The terms
+.\" * are also available at https://curl.se/docs/copyright.html.
+.\" *
+.\" * You may opt to use, copy, modify, merge, publish, distribute and/or sell
+.\" * copies of the Software, and permit persons to whom the Software is
+.\" * furnished to do so, under the terms of the COPYING file.
+.\" *
+.\" * This software is distributed on an "AS IS" basis, WITHOUT WARRANTY OF ANY
+.\" * KIND, either express or implied.
+.\" *
+.\" * SPDX-License-Identifier: curl
+.\" *
+.\" **************************************************************************
+.\"
+.TH CURLOPT_DEBUG_CONNCACHE 3 "31 Oct 2023" libcurl libcurl
+.SH NAME
+CURLOPT_DEBUG_CONNCACHE \- debug the connection cache
+.SH SYNOPSIS
+.nf
+#include <curl/curl.h>
+
+CURLcode curl_easy_setopt(CURL *handle, CURLOPT_DEBUG_CONNCACHE, long onoff);
+.fi
+.SH DESCRIPTION
+This option enables debugging the internal connection cache associated with
+your easy handle. Several debug settings are copied from \fIhandle\fP to every
+associated internal connection cache:
```

Comment:
> I don't quite understand this description: _every connection cache_. There is only one connection cache per easy handle.

Yes but over the lifetime of the easy handle it may be associated with multiple connection caches and they're all going to receive the copied settings. I take your point though here and your other comments, the wording could use some improvement.

Reply:
Since it is still always only one cache at a time I think "the cache" is more apt than "every cache".